phnt/include/ntzwapi.h

```diff
@@ -836,6 +836,26 @@ ZwConvertBetweenAuxiliaryCounterAndPerformanceCounter(
     _Out_ PLARGE_INTEGER ConversionError
     );
 
+#if (PHNT_VERSION >= PHNT_WIN11)
+
+NTSYSCALLAPI
+NTSTATUS
+NTAPI
+ZwCopyFileChunk(
+    _In_ HANDLE SourceHandle,
+    _In_ HANDLE DestinationHandle,
+    _In_opt_ HANDLE EventHandle,
+    _Out_ PIO_STATUS_BLOCK IoStatusBlock,
+    _In_ ULONG Length,
+    _In_ PLARGE_INTEGER SourceOffset,
+    _In_ PLARGE_INTEGER DestOffset,
+    _In_opt_ PULONG SourceKey,
+    _In_opt_ PULONG DestKey,
+    _In_ ULONG Flags
+    );
+
+#endif
+
```

Comment:
Please remove this, see the comment in near the top of the file:
https://github.com/winsiderss/systeminformer/blob/30f4db6992fe4dfb8e0f556945f70b895cb56ee6/phnt/include/ntzwapi.h#L1-L5

Reply:
Oh, sorry